kakao/KakaoLoginIOS/KakaoLoginIOS/AppDelegate.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AppDelegate.swift
+//  KakaoLoginIOS
+//
+//  Created by Kevin Kang on 2017. 9. 15..
+//  Copyright © 2017년 Hara Kang. All rights reserved.
+//
```

Comment:
The header comments in files should have the Apache 2 license, see the other iOS headers in this project for reference.

This is ok to merge without changing, I can fix it in a follow up PR.